index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+    <header>
+        <div class="float-left">
+            <span id="menu"></span>
+            <img class="youtube-logo" src="./image/logo.png" alt=" logo">
+        </div>
+        <form action="www.microverse.com" method="post">
+            <input type="search" name="search" placeholder="Search">
+            <button id="button-search" type="submit">search</button>
+        </form>
+        <div class="float-right">
+            <span id="upload-button"></span>
+            <button id="sign-up-button">Sign up</button>
+        </div>
+    </header>
+
+    <main>
+
+        <aside id="thumbnails-aside">
```

Comment:
The ```<aside>``` is not to be nested in the main. The aside should take the video previews and the main should have the main video together with the other contents